apps/api/src/admin/participant_manager.py

```diff
@@ -52,7 +52,15 @@ async def check_in_applicant(uid: str, associate: User) -> None:
     record: Optional[dict[str, object]] = await mongodb_handler.retrieve_one(
         Collection.USERS, {"_id": uid, "role": Role.APPLICANT}
```

Comment:
Issue: we'll need to update the query to select roles other than just applicant. I think we could just remove the `role` selector all together since we're checking status anyway, and it's not possible for anyone to have reached the final two statuses, but for completeness, `{"role": {"$exists": True}}` could also work.